settings.gradle.kts

```diff
@@ -0,0 +1,27 @@
+pluginManagement {
+    repositories {
+        mavenCentral()
+        gradlePluginPortal()
+        maven("https://oss.sonatype.org/content/repositories/snapshots")
+        maven("https://maven.architectury.dev/")
+        maven("https://maven.fabricmc.net")
+        maven("https://maven.minecraftforge.net/")
+        maven("https://repo.spongepowered.org/maven/")
+        maven("https://repo.sk1er.club/repository/maven-releases/")
+    }
+    resolutionStrategy {
+        eachPlugin {
+            when (requested.id.id) {
+                "gg.essential.loom" -> useModule("gg.essential:architectury-loom:${requested.version}")
+            }
+        }
+    }
+}
+
+rootProject.name = "OneConfigStages"
+
+include("Wrapper")
```

Comment:
maybe lowercase the submodules name

Reply:
I'm changing the name below to the old names, this is just the folder name